Lib/fontTools/cffLib/__init__.py

```diff
@@ -1027,20 +1027,16 @@ def buildConverters(table):
 	return d
 
 
-class BaseConverter(object):
+class SimpleConverter(object):
 
 	def read(self, parent, value):
+		if not hasattr(parent, "file"):
+			return self._read(parent, value)
 		file = parent.file
 		pos = file.tell()
-		try:
```

Comment:
why did you remove the try... finally... block?

Reply:
Now SimpleConverter is the base class from which other Converters are derived, and the default meaningful _read method always exists and it simply returns 'value'. So I removed try finally block.

Reply:
No, you need to put it back. The function of the finally is to ensure the statement after it will be called at the end whether the block inside the try completed successfully or it raised an exception.

Reply:
Uh... try block should exist with considering overridden _read throw any exception ... sorry I will restore the try finally block.

Reply:
yep. thanks